liferay-echo/liferay-portal
# LPS-136338 buildLang

In order to test this:
1. Deploy everything (maybe is better to do an `ant all`).
2.  Execute `scr:enable com.liferay.journal.web.internal.info.collection.provider.BasicWebContentSingleFormVariationInfoCollectionProvider` in gogo shell.
3. Add a `Collection Display` to a content page.
4. Select `Basic Web Content` collection provider.
5. A `Filter Collection` option will appear in the collection selector dropdown (it should be hidden for any other collection).
